lib/store_web/templates/starbook/index.html.eex

```diff
@@ -0,0 +1,60 @@
+  </section>
+
+  <section id="display">
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/hinge.png") %>" />
+    <div>
+      <h3>Matte or glossy—why not both?</h3>
+      <p>The 14-inch ARC display features a protective hard coating that lets color shine through like glass while diffusing ambient light like a matte display. Combined with IPS technology and 400 nits of brightness, the display is visible in nearly any light and angle. Which is convenient, because the 180° hinge lets you get some pretty wild angles.</p>
+    </div>
+  </section>
+
+  <section id="trackpad">
+    <div>
+      <h3>Slip and swipe.</h3>
+      <p>Glide across the 5.5-inch-diagonal glass trackpad for precise and smooth multi-touch gestures. Scroll through documents or web pages, fly through your workspaces, and swipe to go back throughout elementary OS—all with 1:1 gestures that perfectly track your fingers.</p>
+    </div>
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/trackpad.png") %>" />
+  </section>
+
+  <section id="keyboard">
+  <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/keyboard-us.png") %>" />
+  <div>
+    <h3>A keyboard that’s pure function.</h3>
+    <p>Choose a US or UK layout featuring comfortable backlit keys with snappy scissor mechanisms. Tap a single key to switch between F1 through F12 or function keys that control volume, media, brightness, and more. Plus, an extra column of handy keys including Home, End, and Page Up/Down.</p>
+  </div>
+  </section>
+
+  <section id="privacy">
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/privacy.png") %>" />
+    <h3>Ready for remote, with privacy built-in.</h3>
+    <p>The HD webcam is flanked by stereo microphones for clear video and voice calls. The bright white indicator shines any time the camera is on, while elementary OS lets you know any time the mics are being accessed.</p>
+  </section>
+
+  <section id="connectivity">
+    <img alt="StarBook Mk V" loading="lazy" src="<%= Routes.static_path(Endpoint, "/images/devices/starbook/side.png") %>" />
+  <h3>Versatile connectivity.</h3>
+  <p>Wi-Fi 6 and Bluetooth 5.1 for all things wireless. USB-C with Thunderbolt 4, power delivery, and display support for charging, accessories, output all in one. Full-size USB 3 on each side. HDMI for easy output virtually everywhere. Combination headphone/microphone jack. And a microSD card slot for expansion, mobile device file transfers, or flashing maker projects—you name it.</p>
```

Comment:
This seems a littler text heavy to me where we probably want this to be more of a list or a grid. I wonder if we can use like bullets instead where each bullet is an icon like a little wifi icon, bluetooth icon, etc.

Let me know if you need help with icons or a sketch or something would help

Reply:
I have an initial improvement but icons would be welcome if you'd want to add them.

![Screenshot from 2021-12-04 00 59 42](https://user-images.githubusercontent.com/611168/144702332-1a67a886-5707-48f6-8c93-c242f5e18f8a.png)